xmtp_mls/src/groups/mod.rs

```diff
@@ -423,13 +423,16 @@ where
                 &provider,
                 &context.identity,
                 &group_config,
-                GroupId::from_slice(group_id),
+                GroupId::from_slice(existing_group_id),
             )?
         } else {
             OpenMlsGroup::from_creation_logged(&provider, &context.identity, &group_config)?
         };
 
         let group_id = mls_group.group_id().to_vec();
+        // If not an existing group, the creator is a super admin and should publish the commit log
+        // Otherwise, for existing groups, we'll never publish the commit log until we receive a welcome message
+        let should_publish_commit_log = existing_group_id.is_none();
```

Comment:
@richardhuaaa updated to mark groups created from backup as `false` for `should_publish_commit_log`. If our client is in fact a super admin this should get updated in our db when we process a welcome for that group.